docs/src/main/asciidoc/Bootable_Guide.adoc

```diff
@@ -28,140 +28,41 @@ ifdef::basebackend-html[toc::[]]
-        </execution>
-      </executions>
-    </plugin>
-  </plugins>
-</build>
-----
-
-The next chapter covers the plugin configuration items that are required to identify the WildFly server version and content.
-
-[[wildfly-jar-maven-plugin-galleon-configuration]]
-== Galleon configuration
-
-The Bootable JAR Maven plugin depends on https://docs.wildfly.org/galleon/[Galleon] to construct the WildFly server contained in the JAR file. 
-
-Galleon is configured thanks to the Maven plugin ```<configuration>``` element.
-
-The first required piece of information that Galleon needs is a reference to the WildFly Galleon feature-pack. 
-The WildFly Galleon feature-pack is a maven artifact that contains everything needed to dynamically provision a server. 
-This feature-pack, as well as the feature-packs on which its depends,
-are deployed in public maven repositories. 
-
-When the bootable JAR Maven plugin builds a JAR, WildFly feature-packs are retrieved and 
-their content is assembled to create the server contained in the JAR.
-
-Once you have identified a WildFly Galleon feature-pack, you need to select a set of 
-link:#wildfly_layers[WildFly Layers] that are used to compose the server.
-
-The set of Galleon layers to include is driven by the needs of the application you are developing.
-The list of link:#wildfly_layers[WildFly Layers] provides details on 
-the server features that each layer brings. Make sure that the API and server features you are using (eg: Jakarta RESTful Web Services, MicroProfile Config, datasources)
-are provided by the layers you are choosing.
-
-If you decide not to specify Galleon layers, a server containing all MicroProfile subsystems is
-provisioned. (The server configuration is identical to the ```standalone-microprofile.xml``` configuration
-in the traditional WildFly server zip.)
-
-Maven Plugin configuration extract example:
-
-[source,xml]
-----
-<build>
-  <plugins>
-    <plugin>
-      <groupId>org.wildfly.plugins</groupId>
-      <artifactId>wildfly-jar-maven-plugin</artifactId>
-      <configuration>
-        <feature-pack-location>wildfly@maven(org.jboss.universe:community-universe)</feature-pack-location> (1)
-        <layers>
-          <layer>jaxrs-server</layer> (2)
-        </layers>
-      </configuration>
-      <executions>
-        <execution>
-          <goals>
-            <goal>package</goal>
-          </goals>
-        </execution>
-      </executions>
-    </plugin>
-  </plugins>
-</build>
-----
-
-(1) In this plugin configuration extract, we are retrieving the latest WildFly Galleon feature-pack installed in the
- ```org.jboss.universe:community-universe``` Galleon universe. In case you would like to provision a specific version of the server,
-you would need to specify the server version, for example ```wildfly@maven(org.jboss.universe:community-universe)#21.0.0.Final```
-
-(2) The link:#gal.jaxrs-server[jaxrs-server layer] and all its dependencies are provisioned.
-
 [NOTE]
```

Comment:
Sorry I missed this before, IMO this should be either `[WARNING]` or `[CAUTION]`.

Reply:
Thank-you, fixed